### DISCUSSION TITLE
school password

### DISCUSSION THREAD

**Discussion by vernal-ria0** [3kh0/ext-remover#511] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I successfully unenrolled but now the netlog trick wont work since there are no policies. any other method

**Comment by obkduishinijf** [comment, marked as answer]
enroll back in, get the wifi password, write it down, unenroll, silly goose, netlog only works because the password is hidden in the policy.

**Reply by obkduishinijf** [reply]
@vernal-ria0 mark as answer, it's been 2 days. if it didn't help, dm me on discord, sniper#2638 and i'll guide you through it
